twitter/View Controllers/TimelineViewController.m

```diff
@@ -8,8 +8,15 @@
 
 #import "TimelineViewController.h"
 #import "APIManager.h"
+#import "AppDelegate.h"
+#import "LoginViewController.h"
+#import "Tweet.h"
+#import "TweetCell.h"
 
-@interface TimelineViewController ()
+@interface TimelineViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic, strong) NSMutableArray<Tweet *> *arrayOfTweets;
```

Comment:
Nice use of lightweight generics! Because the signature is so readable the name can just be `tweets`.